<?xml version="1.0"?>
<launch>

  <!-- send parameters to parameter server -->
  <rosparam command="load" ns="room_exploration" file="$(find ipa_room_exploration)/ros/launch/room_exploration_action_server_params.yaml"/>

  <!--  -->
  <node ns="room_exploration" pkg="ipa_room_exploration" type="room_exploration_server" name="room_exploration_server" output="screen" respawn="true" respawn_delay="2"/>

</launch>
